canonical-web-and-design/maas-ui
# fix(ui): fix the scripts sorting

## Done

- Alphabetically sort the default commissioning scripts.
- Update files to typescript.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- On the machine list select some machines and choose commission from the take action menu.
- In the form check the default commissioning scripts are sorted alphabetically.
- In the settings visit the list of commissioning scripts.
- Check the default commissioning scripts are sorted alphabetically.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1667
